src/loader.ts

```diff
@@ -131,18 +131,12 @@ export async function getUserFunction(
         }
       }, functionModule);
 
-    // TODO: do we want 'function' fallback?
     if (typeof userFunction === 'undefined') {
-      // eslint-disable-next-line no-prototype-builtins
-      if (functionModule.hasOwnProperty('function')) {
-        userFunction = functionModule['function'];
-      } else {
-        console.error(
-          `Function '${functionTarget}' is not defined in the provided ` +
-            'module.\nDid you specify the correct target function to execute?'
-        );
-        return null;
-      }
+      console.error(
```

Comment:
Does this impact the case where no `--target` flag or `FUNCTION_TARGET` env var is provided?

I think we want to still keep the default behavior documented in the README for that, where we look for a function named `function`

i.e. running just `functions-framework` starts up a function named `function` still

However if someone runs `functions-framework --target=myFn` and `myFn` does **not** exist we should error instead of starting up a function named `function`, because that's a conflict with the user's specifications.

Reply:
It shouldn't affect either case, the target / FUNCTION_TARGET [provides 'function' as a default value](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/src/options.ts#L86-L90) so when no value is provided it will fallback to the expected default behavior. 

We have some test coverage [here](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/test/options.ts#L50).